### PR TITLE
browse solution vertical slice restructure

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     "nuke": "npm run clean && rm -rf node_modules/",
     "postinstall": "npm run build",
     "start": "babel-node server",
-    "test:integration": "NODE_ENV=development babel-node integration-tests/testcafeRunner.js",
-    "test:integration_win": "babel-node integration-tests/testcafeRunner.js",
+    "test:integration": "NODE_ENV=development babel-node testcafeRunner.js",
```
Comment (alibombaye) [inline package.json]: any reason why we are moving the testcafeRunner into integration-tests folder?

Comment (kat1906) [reply package.json]: moving it out! it was the only thing in there

Comment (alibombaye) [reply package.json]: 👍

Comment (alibombaye) [reply package.json]: ah sorry I got my reds and greens mixed up. What you've done is what I expected LOL

**Pull request merged**